data-visualization/data-visualization.md

```diff
@@ -1,7 +1,7 @@
 # Data Visualization
 
 ### Projected Time
-30-45 minutes
+_ minutes
```

Comment:
I changed the time here since I added a new activity under `Challenge` that might add a little bit more time to this projection.

Reply:
Reading through the lesson plan + activities this seems like a full-day. Does that seem about right?

Reply:
Sounds good to me; made the change below